### ISSUE TITLE
pod 'Google-Mobile-Ads-SDK'

### ISSUE THREAD

**Issue by icoco** [googleads/googleads-mobile-ios-examples#57]
the document said use 'pod 'Google-Mobile-Ads-SDK'

but the code is not , that use 'firebase' , cannot understand what's it

**Comment by ericleich** [comment]
tl;dr You can use `Google-Mobile-Ads-SDK` or `Firebase/AdMob`.

AdMob can also be used [with Firebase](https://firebase.google.com/docs/admob/ios/quick-start), so what is in the example still works. With the sample, `Firebase/AdMob` effectively downloads `Google-Mobile-Ads-SDK` as well as the `Firebase/Analytics` pod.

Having said that, we moved the bulk of our developer docs to the [standalone AdMob](https://developers.google.com/admob/ios/quick-start) from the Firebase site a couple months ago, so these samples should also strip out Firebase to be consistent with the standalone AdMob experience. Will leave this ticket open to address this.

**Comment by stowy** [comment]
Commit [#21544dd5](https://github.com/googleads/googleads-mobile-ios-examples/commit/21544dd5291964b7cca014f04f925fe54e9bd4a5) removes references to Firebase and updates podfiles to use `Google-Mobile-Ads-SDK`, [release 6.12](https://github.com/googleads/googleads-mobile-ios-examples/releases/tag/6.12) has been created with this update. 

Marking this issue as closed as a result of these changes. Thanks for the feedback!
